src/calculateRentalCost.js

```diff
@@ -5,6 +5,22 @@
  */
 function calculateRentalCost(days) {
   // write code here
+
+  const longTerm = 7;
+  const longTermDiscount = 50;
+  const mediumTerm = 3;
+  const mediumTermDiscount = 20;
+  const calc = days * 40;
+
+  if (days >= longTerm) {
+    return calc - longTermDiscount;
+  }
+
+  if (days >= mediumTerm) {
+    return calc - mediumTermDiscount;
+  }
```

Comment:
The logic for applying discounts is incorrect. Currently, if `days` is greater than or equal to `longTerm`, the function returns immediately, applying only the long-term discount. This means the medium-term discount will never be applied if the long-term condition is met. Consider restructuring the logic to ensure that the appropriate discount is applied based on the number of days.

Reply:
you mentioned " This means the medium-term discount will never be applied if the long-term condition is met." this is what we were asked for. Maybe in another discount program that logic wouldn't work, however I don't see a reason to reject the logic for this discount program.